pkg/operation/care/constraints.go

```diff
@@ -252,6 +254,10 @@ func (c *Constraint) CheckForProblematicWebhooks(ctx context.Context) (gardencor
 	}
 
 	for _, webhookConfig := range validatingWebhookConfigs {
+		if strings.Contains(webhookConfig.Annotations[resourcesv1alpha1.OriginAnnotation], c.shoot.SeedNamespace) {
+			continue
+		}
```

Comment:
This is not required as we will get only that webhook here that doesn't have `resources.gardener.cloud/managed-by` label.